e2e/testdata/fn-eval/fn-source/exclude-meta-resources-by-default-out-of-place/.expected/exec.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
```

Comment:
harness should set this automatically. Can be a follow up.

Reply:
I suspect this can only be set in the script, unless we use a wrapper script.